docs/csharp/language-reference/keywords/interface.md

```diff
@@ -9,31 +9,40 @@ ms.assetid: 7da38e81-4f99-4bc5-b07d-c986b687eeba
 ---
 # interface (C# Reference)
```

Comment:
Should "interface" be added to no-loc in the metadata?

Reply:
It looks that H1 (or C# keywords) are treated in some special way, for example:
https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/keywords/interface

Also note that the C# keywords are not translated in the TOC.
So, I guess some "magic" is applied to the C# keywords met in the titles (and, thus, in TOCs)

Reply:
Yes it seems that it doesn't get localized in the heading. But in the text under the H1 it gets localized, which I think not desirable.

Reply:
If it's used as a keyword, it is usually code-fenced and not localized. However, when it's used as a term, it should be localized and not code-fenced.

Reply:
So, maybe, in this particular case, the title should be localized as well.

Reply:
This shouldn't be localized. I'll add it to the metadata.